### PR TITLE
Add my recent works  ,skills tech  and contact form sections of the mobile version   

### PR DESCRIPTION
**Pull request by youmari**
A project and contact form section of the mobile version

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -81,6 +81,330 @@ header {
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background-color: #36b37e;
+  border-radius: 4px;
+  margin-top: 29px;
+  cursor: pointer;
+}
```
Comment (mikethreels) [inline css/style.css:201]: - [ ] Great job on the button 👍 however according to the requirements you should also implement that the button changes colour when you hover over it.

File: index.html [hunk 2]
```diff
@@ -45,6 +45,170 @@ <h2 class="secondary-titel">
...
+          </div>
+          <form action="#" method="get">
+            <button type="submit" class="project-btn">See Project</button>
+          </form>
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="main-about-me-container">
+        <h1 class="about-me-titel">About me</h1>
```
Comment (mikethreels) [inline index.html:140]: - [ ] The `<h1>` tag is already in use please us please us a header tag but with another number.

File: index.html [hunk 3]
```diff
@@ -45,6 +45,170 @@ <h2 class="secondary-titel">
...
+      <div class="projects-container">
+        <div class="project1 projects-cards">
+          <img class="project-img" src="#" alt="project1-image">
+          <div class="project-titel-container">
+            <h4 class="project-titel">My Portfolio</h4>
+          </div>
+          <div class="tags-container">
+            <div class="html-tag tags">html</div>
+            <div class="css-tag tags">css</div>
+          </div>
```
Comment (mikethreels) [inline index.html:64]: - [ ] Good job on these lists 💪 however the list of project technologies should also be inside an `<ul>` tag.

File: index.html [hunk 4]
```diff
@@ -45,6 +45,170 @@ <h2 class="secondary-titel">
           <a href="https://www.github.com/youmari" class="github-link"><img src="./images/github.svg" alt="github icon"></a>
```
Comment (SajjadAhmad14) [inline index.html:45]: [ OPTIONAL ] Please make sure all linter checks are passing before asking for a review.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Yassine Filali Omari, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL
- [ ] Please check inline comments
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**